src/biotite/structure/io/pdbx/convert.py

```diff
@@ -111,7 +111,7 @@ def _filter(category, index):
 
 
 def get_sequence(pdbx_file, data_block=None):
-    """
+    """""
```

Comment:
I just wanted to merge, then I saw this typo:
```suggestion
    """
```